### PR TITLE
Enable wasmtime caching for PVF (companion for #8057)

### PR DESCRIPTION
**Pull request by pepyakin**

### CODE REVIEW

File: node/service/src/lib.rs [hunk 1]
```diff
@@ -1005,37 +1005,50 @@ pub fn build_full(
 	grandpa_pause: Option<(u32, u32)>,
 	jaeger_agent: Option<std::net::SocketAddr>,
 ) -> Result<NewFull<Client>, Error> {
+	let isolation_strategy = {
+		#[cfg(not(any(target_os = "android", target_os = "unknown")))]
+		{
+			let cache_path = config.database.path();
```
Comment (pepyakin) [inline node/service/src/lib.rs:1011]: This is basically the same approach we've taken in the av-store.

Comment (ordian) [reply node/service/src/lib.rs:1011]: This is actually the parent directory of the cache path, right? Maybe worth renaming it to `cache_base_path` or something like that.

Comment (pepyakin) [reply node/service/src/lib.rs:1011]: this is actually a good idea, thanks

**Comment by pepyakin** [comment]
bad bot

bot merge

**Review by bkchr — APPROVED** [review]

**Pull request merged**